### ISSUE TITLE
Crash on ubuntu with anki 2.54

### ISSUE THREAD

**Issue by winstonDeGreef** [stevetasticsteve/Anki_bulk_importer#9]
I tried to run the add on, but got the following error.
```
Anki 2.1.54 (b6a7760c) Python 3.9.10 Qt 6.3.1 PyQt 6.3.1
Platform: Linux
Flags: frz=True ao=True sv=2
Add-ons, last update check: 2022-09-04 18:29:47
Add-ons possibly involved: ⁨Bulk importer⁩

Caught exception:
Traceback (most recent call last):
  File "/home/winston/.local/share/Anki2/addons21/1312111882/__init__.py", line 17, in CLAImporter
    bulk_importer.run()
  File "/home/winston/.local/share/Anki2/addons21/1312111882/bulk_importer.py", line 357, in run
    Window(aqt.mw)
  File "/home/winston/.local/share/Anki2/addons21/1312111882/bulk_importer.py", line 88, in __init__
    self.pictureTable = self.draw_table(
  File "/home/winston/.local/share/Anki2/addons21/1312111882/bulk_importer.py", line 142, in draw_table
    self.table.setSelectionBehavior(self.table.SelectRows)
AttributeError: 'QTableView' object has no attribute 'SelectRows'
```

**Comment by stevetasticsteve** [comment]
I see that you're using the QT6 version of Anki. I've not updated the add-on to run on QT6 yet.

I will try get round to looking at that in the next few months. 

Until then unfortunately I can only recommend installing the QT5 version of Anki. 

Thanks for letting me know

**Comment by randaalex** [comment]
The same behaviour on macos: 
And qt5 version doesn't exist for apple silicon.

```
Error
An error occurred. Please start Anki while holding down the shift key, which will temporarily disable the add-ons you have installed.
If the issue only occurs when add-ons are enabled, please use the Tools > Add-ons menu item to disable some add-ons and restart Anki, repeating until you discover the add-on that is causing the problem.
When you've discovered the add-on that is causing the problem, please report the issue to the add-on author.
Debug info:
Anki 2.1.61 (0c1eaf4c) Python 3.9.15 Qt 6.3.2 PyQt 6.3.1
Platform: macOS-13.3.1-arm64-arm-64bit
Flags: frz=True ao=True sv=2
Add-ons, last update check: 2023-04-28 14:38:16
Add-ons possibly involved: ⁨Bulk importer⁩

Caught exception:
Traceback (most recent call last):
  File "/Users/ori/Library/Application Support/Anki2/addons21/1312111882/__init__.py", line 17, in CLAImporter
    bulk_importer.run()
  File "/Users/ori/Library/Application Support/Anki2/addons21/1312111882/bulk_importer.py", line 357, in run
    Window(aqt.mw)
  File "/Users/ori/Library/Application Support/Anki2/addons21/1312111882/bulk_importer.py", line 88, in __init__
    self.pictureTable = self.draw_table(
  File "/Users/ori/Library/Application Support/Anki2/addons21/1312111882/bulk_importer.py", line 142, in draw_table
    self.table.setSelectionBehavior(self.table.SelectRows)
AttributeError: 'QTableView' object has no attribute 'SelectRows'
```

**Comment by stevetasticsteve** [comment]
@randaalex thanls for the reminder, @winstonDeGreef so sorry this took so long, I forgot!

I've updated the addon to be compatible with both qt5 and qt6, It should work for you guys. I've tested qt5 version on Ubuntu and qt6 on Windows, I'm unable to test on an Apple device, but I'm sure it will work as well as the other platforms.

The package has been updated on AnkiWeb and you'll be able to update
